Makefile

```diff
@@ -370,6 +370,7 @@ version: mods/ra/mod.yaml mods/cnc/mod.yaml mods/d2k/mod.yaml mods/ts/mod.yaml m
 docs: utility mods version
 	@mono --debug OpenRA.Utility.exe all --docs > DOCUMENTATION.md
```

Comment:
If we're going to be shipping multiple API files with the installer then this should be renamed to TRAIT-API.md.